courses/utils.py

```diff
@@ -26,6 +27,10 @@ def is_grade_valid(override_grade: float):
     return 0.0 <= override_grade <= 1.0
 
 
+def is_letter_grade_valid(letter_grade: str):
+    return re.match("^[A-F]$", letter_grade) is not None
```

Comment:
Could you please have a look at this [comment](https://github.com/mitodl/mitxonline/issues/1762#issuecomment-1649697708) by @arslanashraf7 and make the relevant changes?

Reply:
This PR doesn't try to solve the inconsistency of the letter grades in general. This PR just makes letter grade required when running this management command.